CHANGELOG.md

```diff
@@ -1,3 +1,54 @@
+<a name="3.0.0"></a>
+
+# 3.0.0 (2017-12-01)
+
+### Features
+* **cli:** Added a boat-load of new CLI configuration options. Every option available
+           in the `angular-playground.json` file is now also available as a CLI argument
+           (except @angular/cli arguments). [Read more about it in our docs](http://angularplayground.it/docs/api/configuration).
+           ([9dc1066](https://github.com/SoCreate/angular-playground/pull/68/commits/9dc1066))
```

Comment:
The base URLS on all of these should be https://github.com/SoCreate/angular-playground/commit/ and not https://github.com/SoCreate/angular-playground/pull/68/commits